src/test/java/SYSC4806/Model/CategoryTest.java

```diff
@@ -35,29 +35,14 @@
      */
     @Before
     public void setUp() throws Exception {
-        List<LearningOutcome> lo1 = new ArrayList<>();
-        List<LearningOutcome> lo2 = new ArrayList<>();
-
-        lo1.add(new LearningOutcome());
-        lo1.add(new LearningOutcome());
-        lo1.add(new LearningOutcome());
-        lo1.add(new LearningOutcome());
-        lo2.add(new LearningOutcome());
-        lo2.add(new LearningOutcome());
-
-        Category c1 = new Category("Category1", lo1);
-        Category c2 = new Category("Category2", lo2);
-
-        categoryRepository.save(c1);
-        categoryRepository.save(c2);
     }
 
     /**
      * Deleting the saved information for categories
      */
     @After
     public void tearDown(){
-        categoryRepository.deleteAll();
```

Comment:
Sorry, can you just elaborate on removing the stuff in the tearDown? Is the data persistent now, which is why we are choosing not to clean up the DB? 

Arguably, I think our test cases can use some work. I think we should keep the test values and the actual data separate. Meaning, we would never actually query something like "Programming" as a category because it might actually be used in production. I think the proper way would actually be to insert test data, query the data and assert, and remove all the test data. That's just my two cents though.

Reply:
I removed it because it was originally there when data was being added in setup. 

I agree that the test cases should be improved, but I didn't really want to handle that in this pull request as that wasn't the intention of my changes. Mine were more for the mappings fix which is why I only fixed the tests to the extent that it would build properly. We should create a new issue to fix testing as I agree with your proposition.

Reply:
I think at this stage, the only data we have is being used for testing, because we have no actual functionality to add or remove. So, for this milestone, I think it makes sense to keep our tearDowns nice and small, less to sort through when getting the basic stuff working.

That being said in the future I think it makes a lot of sense to upload our own test database and then clean the DB once the tests are done. Once we are comfortable reading from files to populate the database it'll be nice and easy hopefully.

Reply:
Fair enough, made an issue in #41.